fbrp/examples/10_ifttt/trigger_event_detector.py

```diff
@@ -0,0 +1,19 @@
+import a0
+import time
+
+EVENT_TOPIC = a0.cfg(a0.env.topic(), "/event_topic", str)
+print(f"{EVENT_TOPIC} initialized")
+print(type(EVENT_TOPIC))
+
+i = 0
+p = a0.Publisher(f"{EVENT_TOPIC}")
+print(f"we have a publisher")
+
+while True:
+    print(f"updating config")
+    a0.update_configs()
```

Comment:
What does this update?
There are no cfgs used in this loop. You can drop this line.